src/scripts/modules/media/body/content-baked.less

```diff
@@ -122,36 +143,83 @@ h1.example-title .text {
 
 // Helper mixin for expanding all the selectors
 .make-block(@type) {
+  #blockish>.style(@type);
```

Comment:
It looks like this is copy/pasta'd from somewhere else. Can the code be organized so that this is written in one place and called in 2 places?